docs/pages/telemetry.mdx

```diff
@@ -0,0 +1,60 @@
+---
+title: Telemetry
+sidebar_label: Telemetry
+---
+
+In this section, you will find a description of the DevPod telemetry - why we are collecting telemetry, what data points are we gathering, where we are sending the data, and how to opt-out.
+
+
+### Why do we collect telemetry
+
+Because DevPod is a freely available open source project, we as maintainers have a very limited idea of how the project is being used, and very limited possibilities to gather this information from the users. Without reliable information, it is difficult to make decisions about the prioritization of features, test automation or bug fixes. Deprecation of the flags and features turns into guesswork, and removal becomes nearly impossible.
+To move forward quickly, and ensure long-term maintainability, we will be making decisions about new features, deprecations, prioritizing test coverage, etc., and we want these decisions to be data-driven.
+
+
+### What are we collecting and how
+
+When a user executes a non-trivial CLI command, one that loads the configuration, the CLI will send an event about the command and basic information about the execution environment (Operating system, CPU architecture, DevPod version, ect.). Another event is sent if a command fails, or when it takes longer than 10 seconds to complete. The events are sent to our self-hosted backend which will store them in the database.  
```

Comment:
"which will store them in the database" might be an uncesessary detail :D

Reply:
True. I mainly trying to say that we are not using some third-party provider of analytics services :) 
I'll remove that bit if I will be making other changes to the PR too.

Reply:
ahhh that makes sense. I think you already stated this with "self-hosted backend"